### ISSUE TITLE
The NPM Package for Version 2.2.4 Seems to be Corrupted

### ISSUE THREAD

**Issue by derknorton** [finwo/supercop#7]
I get the following error when trying to install it:
```
$ npm install
npm WARN tarball tarball data for supercop@https://registry.npmjs.org/supercop/-/supercop-2.2.4.tgz (sha512-f5kQA4Lip238qda86oyNQicDbLauXe6iobi2ihBOmhZT6ygQqNVjRAJP7HCrZjZM+Y0dwta34ceY4dEXaTZDsQ==) seems to be corrupted. Trying again.
npm WARN tarball tarball data for supercop@https://registry.npmjs.org/supercop/-/supercop-2.2.4.tgz (sha512-f5kQA4Lip238qda86oyNQicDbLauXe6iobi2ihBOmhZT6ygQqNVjRAJP7HCrZjZM+Y0dwta34ceY4dEXaTZDsQ==) seems to be corrupted. Trying again.
npm ERR! code EINTEGRITY
npm ERR! sha512-f5kQA4Lip238qda86oyNQicDbLauXe6iobi2ihBOmhZT6ygQqNVjRAJP7HCrZjZM+Y0dwta34ceY4dEXaTZDsQ== integrity checksum failed when using sha512: wanted sha512-f5kQA4Lip238qda86oyNQicDbLauXe6iobi2ihBOmhZT6ygQqNVjRAJP7HCrZjZM+Y0dwta34ceY4dEXaTZDsQ== but got sha512-s8X0zLGgSjdmMTNApT7pzLoVJYAc3uUro1KM6aDjW0pPkEV2lJUuRoNORC5nwlPmgOP0TZ5UXsf8aXPGCoVzBg==. (92657 bytes)
```

**Comment by finwo** [comment]
Odd, as I've automated the release whenever a new tag hits the master branch.. For now I'll assume this is due to a hiccup in the networking between the github action runner and npm.

Will re-release 2.2.5

**Comment by finwo** [comment]
Seemed like the release wasn't fully finished, as version 2.2.3 wasn't marked as deprecated either.

Just released 2.2.5 to fix this. Previous versions have automatically been marked as deprecated.

**Comment by derknorton** [comment]
Thanks for your quick update, I just confirmed that 2.2.5 works great!
